auth/src/main/java/com/hms/lib/commonmobileservices/auth/AuthService.kt

```diff
@@ -54,19 +55,18 @@ interface AuthService {
     fun updatePasswordwPhone(password: String?, verifyCode: String?): Work<Unit>
     fun getCode(var1: String?): Work<Unit>
     fun getCodePassword(var1: String?): Work<Unit>
-    fun getPhoneCode(var1: String?, var2: String?): Work<Unit>
+    fun getPhoneCode(var1: String?, var2: String?, activity: Activity): Work<Unit>
```

Comment:
Could you give the parameter names more descriptively instead of var1 or var2?

Reply:
resolved.

Reply:
not resolved. please give the parameter names more clearly instead of var1 value.